src/adapters/marketplace/v00.js

```diff
@@ -39,10 +39,15 @@ class V00_MarkeplaceAdapter {
 
   async createListing(
     ipfsBytes,
-    { deposit = '0', arbitrator },
+    { deposit = '0', arbitrator, commission },
     confirmationCallback
   ) {
     const from = await this.contractService.currentAccount()
+    deposit = commission && commission.amount || '0'
+
+    if (deposit !== '0') {
+      deposit = (Number(deposit)*10**18).toString()
```

Comment:
In general I think it's not safe to use JS Number for such large values... Let's use BigNumber everywhere ?

Either by using the [Big Number](https://www.npmjs.com/package/big-number) package directly (we'll need to add it to package.json) or via https://web3js.readthedocs.io/en/1.0/web3-utils.html#tobn - I'd lean towards the former.